src/config/services/logging.md

```diff
@@ -5,7 +5,7 @@
 The default installation comes with no syslog daemon, there are
 different implementations available.
 
-`socklog` is the implementation from the
+[socklog](http://smarden.org/socklog/) is the implementation from the
 [runit(8)](https://man.voidlinux.eu/runit.8) author and Void Linux
```

Comment:
missed a .eu here

Reply:
Oops! Caught a few more. The only ones left are in installation/downloading.md and are fixed upstream.